azure-pipelines.yml

```diff
@@ -39,4 +39,14 @@ jobs:
     condition: succeededOrFailed()
     inputs:
       testResultsFiles: '**/test-*.xml'
-      testRunTitle: 'flake8-annotations Lint & Test Results'
\ No newline at end of file
+      testRunTitle: 'flake8-annotations Lint & Test Results'
+
+  - script: python3 -m pytest --cov-report xml:cov.xml --cov=flake8_annotations testing/
```

Comment:
Does an xml report also print to normally to stdout? Isn't `--cov-report term-missing` needed for that? The reason it should be printed to stdout is because Azure has failed before in showing published reports so the console output is a fallback.

Reply:
This also applies to the flake8 output if it isn't already being shown in stdout. If you're generating XML for flake8 then the only way I've found to get both XML and stdout is to simply run flake8 twice unfortunately.

Reply:
Added terminal output for code coverage and removed XML reporting for the linter, it's close to useless since you have to click on each error just to see where it is.